grails-app/conf/Config.groovy

```diff
@@ -187,7 +187,9 @@ environments {
 /**
  * API & CORS config
  */
-cors.url.pattern = ['/api/*', '/contact/send', '/profile/*', '/logout/*', '/login/*', '/j_spring_security_check', '/canvas', '/logout', '/j_spring_security_logout']
```

Comment:
Is this ok? I don't think that anything else is needed here.

Reply:
@hpihkala Are views grails-app/views/canvas/embed.gsp and grails-app/views/canvas/editor.gsp currently in use?

Reply:
@kare No they are not used, in fact they shouldn't even exist. All `gsp`s except the email templates should have been deleted. Sometimes they come back accidentally in merges, that might be the case here too.